storage_service/locations/models/package.py

```diff
@@ -1020,12 +1026,13 @@ def finish_reingest(self, origin_location, origin_path, reingest_location, reing
         # Copy pointer file if exists
         # TODO what do if LOCKSS & uncompressed?
         if self.package_type in (Package.AIP, Package.AIC):
-            reingest_pointer_src = os.path.join(origin_location.relative_path, os.path.dirname(origin_path), 'pointer.xml')
-            reingest_pointer_name = 'pointer.' + self.uuid + '.reingest.xml'
-            reingest_pointer_dst = os.path.join(ss_internal.relative_path, reingest_pointer_name)
-            src_space.move_to_storage_service(reingest_pointer_src, reingest_pointer_name, internal_space)
-            internal_space.move_from_storage_service(reingest_pointer_name, reingest_pointer_dst)
-            reingest_pointer = os.path.join(ss_internal.full_path, reingest_pointer_name)
+            if self.pointer_file_path is not None and os.path.isfile(self.pointer_file_path):
```

Comment:
`self.pointer_file_path` may not exist on this machine.  The pointer file may be on a different machine, and `os.path.isfile` will always return false.  For example, if the package comes from a PipelineFS which is only accessible via SSH. This change should be removed.